FireMotD

```diff
@@ -365,8 +365,7 @@ ExploreMemory () {
     MemTotalB="$(< /proc/meminfo grep MemTotal | awk '{print $2}')"
     MemTotalGB="$(printf "%0.2f\n" "$(bc -q <<< scale=2\;"$MemTotalB"/1024/1024)")"
     ProcpsVersion="$(free --version | awk '{ print $4 }')"
-    VersionComparison 3.3.9 "$ProcpsVersion"
-    
+    VersionComparison 3.3.9 "$ProcpsVersion"    
```

Comment:
Not sure why its flagged as diff. Newline?

Reply:
Removed blank line there i think